pgbedrock/privileges.py

```diff
@@ -67,7 +68,8 @@ def analyze_privileges(spec, cursor, verbose):
                                                  desired_items=desired_items,
                                                  dbcontext=dbcontext,
                                                  schema_writers=schema_writers,
-                                                 personal_schemas=personal_schemas)
+                                                 personal_schemas=personal_schemas,
+                                                 restricted_items=restricted_items_this_obj)
```

Comment:
The tests show restricted_items as a set but unless my reading is way off it's going to be a *dict* here.

I suspect that when it's passed to difference_update that it's doing the right thing, but our tests aren't enforcing that - I don't think?

Reply:
Agreed with the except part, I think that makes more grammatical sense.  I also updated the readme to include this use case.